src/providers/vk.js

```diff
@@ -0,0 +1,32 @@
+export default (options) => {
+  const apiVersion = options.apiVersion || '5.126' // https://vk.com/dev/versions
```

Comment:
In an ongoing effort (#1022), we try to reduce the API surface to a common minimum, so we have a consistent API across all providers. I understand that this would make it easier to define the different URLs a tiny bit simpler, but I kindly ask you to remove this, and let this be handled in user land.

Example:
```js
// /api/auth/[...nextauth].js
const apiVersion = "5.126"
...
providers: [
  Providers.VK({
    accessTokenUrl: `https://oauth.vk.com/access_token?v=${apiVersion}`,
    requestTokenUrl: `https://oauth.vk.com/access_token?v=${apiVersion}`,
    authorizationUrl:
      `https://oauth.vk.com/authorize?response_type=code&v=${apiVersion}`,
    profileUrl: `https://api.vk.com/method/users.get?fields=photo_100&v=${apiVersion}`,
  })
]
...
``` 

This will make it much easier to create consistent TypeScript types later on, as well as keeps the documentation simple, and the users will be less confused about the available provider options.

You could instead add an explanation to `www/docs/providers/vk.md` to the VK provider users, explaining how they can override the default `5.126` version (probably also worth mentioning to them, which version is used by default.)

Reply:
Good point! Left `apiVersion` hardcoded and added an example with version override to docs.